tests/local.rs

```diff
@@ -332,162 +381,164 @@ async fn resolve_execute_async_value(mut c: Client, port: u16) -> Result<(), err
 
 mod firefox {
     use super::*;
+    use common::*;
```

Comment:
Hmm, why do we need this `use` now? If it's because of something we use in the macro, we can just stick the `use` in the macro directly instead.